src/topp/HighResPrecursorMassCorrector.cpp

```diff
@@ -71,8 +71,9 @@ using namespace std;
 
   This tool performs precursor m/z correction on picked (=centroided) high resolution data.
 
-  Two methods are available: 'nearest_peak' and 'feature'. Both can be used in order.
+  Three methods are available: 'nearest_peak', 'highest_intensity_peak' and 'feature'. They can be used in order.
```

Comment:
maybe its not clear how the order is and what the implications of combining 'nearest_peak', 'highest_intensity_peak' are